test/md2jira.js

```diff
@@ -4,105 +4,209 @@ var fs = require('fs');
-        jira.should.eql("{code}\nso _no_ further *formatting* is done here\n{code}");
-    });
-    it('should convert language-specific code blocks properly', function() {
-        var jira = j2m.to_jira("```javascript\nvar hello = 'world';\n```");
-        jira.should.eql("{code:javascript}\nvar hello = 'world';\n{code}");
-    });
-    it('should convert unnamed links properly', function() {
-        var jira = j2m.to_jira("<http://google.com>");
-        jira.should.eql("[http://google.com]");
-    });
-    it('should convert named links properly', function() {
-        var jira = j2m.to_jira("[Google](http://google.com)");
-        jira.should.eql("[Google|http://google.com]");
-    });
-    it('should convert headers properly', function() {
-        var h1 = j2m.to_jira("# Biggest heading");
-        var h2 = j2m.to_jira("## Bigger heading");
-        var h3 = j2m.to_jira("### Big heading");
-        var h4 = j2m.to_jira("#### Normal heading");
-        var h5 = j2m.to_jira("##### Small heading");
-        var h6 = j2m.to_jira("###### Smallest heading");
-        h1.should.eql("h1. Biggest heading");
-        h2.should.eql("h2. Bigger heading");
-        h3.should.eql("h3. Big heading");
-        h4.should.eql("h4. Normal heading");
-        h5.should.eql("h5. Small heading");
-        h6.should.eql("h6. Smallest heading");
+describe('to_jira', function () {
+  it('should exist', function () {
+    should.exist(j2m.to_jira);
+  });
+
+  it('should be a function', function () {
+    j2m.to_jira.should.be.a('function');
+  });
+
+  describe('emphasis formatting', function () {
+    describe('bold formatting', function () {
+      it('should convert bolds properly', function () {
+        var jira = j2m.to_jira('**bold words**');
+        jira.should.eql('*bold words*');
+      });
+
+      it("should handle multiple bold sections in a line", function () {
+        var jira = j2m.to_jira("**this should be bold** this should not **this should be bold**");
+        jira.should.eql("*this should be bold* this should not *this should be bold*");
+      });
+
+      it("does not perform intraword formatting on asterisks", function () {
+        var jira = j2m.to_jira("a*phrase*with*asterisks");
+        jira.should.eql("a*phrase*with*asterisks");
+      });
+
+      it('handles bolds at the end of sentences', function () {
+        var jira = j2m.to_jira('A sentence ending in **bold**.');
+        jira.should.eql('A sentence ending in *bold*.');
+      });
+
+      it('formats bolds while leaving intraword asterisks untouched', function () {
+        var jira = j2m.to_jira('**bold*phrase*with*internal*asterisks**');
+        jira.should.eql('*bold*phrase*with*internal*asterisks*');
+      });
+
+      // TODO: Fix the code so this test can be unskipped
+      it.skip('does not apply bold formatting without an asterisk pair at the start of the phrase', function () {

```

Comment:
As this PR fixes minor issue, why do you skip this test - was it existing issue?

Reply:
Yes, it was an existing issue. See the Note in the PR description.